salt/metalk8s/addons/volumes/storage-operator.sls.in

```diff
@@ -0,0 +1,10 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+@@ServiceAccount
+---
+@@Role
+---
+@@RoleBinding
+---
+@@Deployment
```

Comment:
Do we have a reason to put yaml in directory `storage-operator/deploy` instead of directly here ?

Reply:
When you ugrade operator-sdk, there is no way to upgrade your generated project.

You need to re-create one from scratch with the new version, do a diff with the new scaffolding and your current code and then apply the changes manually.

Keeping files where they are generated will ease the painful manual process.

Reply:
Also, some of those files are regenerated by `operator-sdk generate` into the `deploy/` folder of your operator, and we don't want to need to manually keep the copy in sync.